### PR TITLE
FeatureFlags: Change alpha/beta language to align with release staging language

### PR DESCRIPTION
**Pull request by ryantxu**
This updates the feature toggle language to better align with our normal release staging language.

This adds a new stage for "private beta" -- previously we just had beta, and I moved them all to "public beta"

### CODE REVIEW

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md
Comment (ryantxu) [inline docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:1]: please make any docs suggestions in the toggles_gen_test.go file, not this file directly

File: pkg/services/featuremgmt/toggles_gen_test.go [hunk 1]
```diff
@@ -254,44 +254,44 @@ weight: 150
 
 # Configure feature toggles
 
-You use feature toggles, also known as feature flags, to turn experimental or beta features on and off in Grafana. Although we do not recommend using these features in production, you can turn on feature toggles to try out new functionality in development or test environments.
+You use feature toggles, also known as feature flags, to enable or disable features in Grafana. You can turn on feature toggles to try out new functionality in development or test environments.
 
 This page contains a list of available feature toggles. To learn how to turn on feature toggles, refer to our [Configure Grafana documentation]({{< relref "../_index.md#feature_toggles" >}}). Feature toggles are also available to Grafana Cloud Advanced customers. If you use Grafana Cloud Advanced, you can open a support ticket and specify the feature toggles and stack for which you want them enabled.
 
-## Stable feature toggles
+## Feature toggles
```
Comment (ryantxu) [inline pkg/services/featuremgmt/toggles_gen_test.go:261]: I kinda hacked this file up to avoid alpha/beta... but it can use some help!

**Comment by ryantxu** [comment]
@timlevett -- can you take a look?  I think this is progress towards using the same language

(and the PRs that touch feature toggles get stale fast as people change them often!)

**Review by timlevett — APPROVED** [review]
LGTM

**Pull request merged**